kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -69,30 +70,32 @@ guDateRange.directive('guDateRange', [function () {
             ctrl.guDateFormat = ctrl.guDateFormat || 'DD-MMM-YYYY';
             ctrl.guAnyTimeText = ctrl.guAnyTimeText || 'anytime';
             ctrl.guFirstDay = ctrl.guFirstDay || 0;
-
             var startInput = el.find('.gu-date-range__input__start--hidden')[0];
             var startContainer = el.find('.gu-date-range__overlay__pikaday--start')[0];
 
             var endInput = el.find('.gu-date-range__input__end--hidden')[0];
             var endContainer = el.find('.gu-date-range__overlay__pikaday--end')[0];
 
             var iso8601Format = 'YYYY-MM-DDTHH:mm:ssZ';
+            const tenYearsInMilliseconds = (10 * 365 * 24 * 60 * 60 * 1000);
+            const tenYearsFromNow =  new Date(Date.now() + tenYearsInMilliseconds);
```

Comment:
Do we not use [moment](http://momentjs.com) anywhere? If not we should, it's great.
[Looks like we install it](https://github.com/guardian/grid/blob/master/kahuna/package.json#L25).